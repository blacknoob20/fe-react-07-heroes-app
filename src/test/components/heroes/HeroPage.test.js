import { mount } from 'enzyme';
import { MemoryRouter } from 'react-router-dom/cjs/react-router-dom.min';
import { HeroPage } from "../../../components/heroes/HeroPage";

describe('Pruebas en el componente <HeroPage/>', () => {
    const history = {
        length: 10,
        push: jest.fn(),
        goBack: jest.fn(),
    };

    const wrapper = mount(
        <MemoryRouter initialEntries={['/hero']}>
            <HeroPage history={history} />
        </MemoryRouter>
    );

    test('Debe mostrarse correctamente el redirect si no hay argumentos en la URL.', () => {
        expect(wrapper).toMatchSnapshot();
        expect(wrapper.find('Redirect').exists()).toBe(true);
    });

});
