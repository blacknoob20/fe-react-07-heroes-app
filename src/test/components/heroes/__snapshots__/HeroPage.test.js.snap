// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente <HeroPage/> Debe mostrarse correctamente el redirect si no hay argumentos en la URL. 1`] = `""`;
